lntest/harness.go

```diff
@@ -927,10 +926,6 @@ type OpenChannelParams struct {
 	// virtual byte of the transaction.
 	SatPerVByte btcutil.Amount
 
-	// ConfTarget is the number of blocks that the funding transaction
-	// should be confirmed in.
-	ConfTarget fn.Option[int32]
```

Comment:
Wait don't we still want to keep this in place? Since on CLI/RPC we accept the fee rate or conf target.

Reply:
This is only used in itest - previously we called `OpenChannel` with empty values and the default conf was used. Then this optional conf target was added so we don't have to change all the callsites in the tests. Since now it's using the default again, we can then remove this.

Reply:
actually gonna keep it - so future change won't need to touch it again.